README.md

```diff
@@ -128,6 +128,7 @@ If the checks don't appear or if the checks are stuck in progress, there's a few
    ```markdown
    _underscore_
    ```
+1. Replace the * with an _ in line 115 of README.md
```

Comment:
Again, I don't think this is the way to go. I think the activity works better by getting the learner to use the Actions log to identify what and where the error is. Maybe a _hints_ detail block could be used to help stuck learners.

The preceding sentence has a typo in it which makes the activity very confusing. This definitately needs to be fixed. Perhaps something like:

1. Update the code in the `ci` branch to get the test to pass. You need to change the incorrect code to look something like this:
   ```markdown
   _underscore_
   ```
   <details><summary>hint</summary>
   You need to change line 115 of <code>README.md</code> to use _ for emphasis insead of *.
   </details>

Reply:
> I think the activity works better by getting the learner to use the Actions log to identify what and where the error is

That's a major learning objective for this course for sure. We moved this to another, shorter file to make it easier to find.

Reply:
Step 2 in README.md
> _By looking at the logs, can you identify which tests failed?_ To find it, go to one of the failed builds and scrolling through the log. Look for a section that lists all the unit tests. We're looking for the name of the test with an "x".

What do you mean by 'the name of the test with an "x"'?